manticore/platforms/evm_world_state.py

```diff
@@ -0,0 +1,405 @@
+import logging
+import os
+from abc import ABC, abstractmethod
+from typing import Dict, Optional, Set, Union
+from urllib.parse import ParseResult, urlparse
+from web3 import Web3  # type: ignore
```

Comment:
I think it would be better to add this to [`mypy.ini`](https://github.com/trailofbits/manticore/blob/master/mypy.ini)

Should look something like
```
[mypy-web3.*]
ignore_missing_imports = True
```

Reply:
Thanks!